dags/glam_fog.py

```diff
@@ -206,24 +197,27 @@
             >> clients_histogram_aggregate
         )
 
-        (
-            clients_scalar_aggregate
-            >> scalar_bucket_counts
-            >> scalar_probe_counts
-            >> probe_counts
-        )
-        (
-            clients_histogram_aggregate
-            >> histogram_bucket_counts
-            >> histogram_probe_counts
-            >> probe_counts
-        )
-        probe_counts >> sample_counts >> extract_probe_counts >> export >> pre_import
-        (
-            clients_scalar_aggregate
-            >> user_counts
-            >> extract_user_counts
-            >> export
-            >> pre_import
-        )
-        clients_histogram_aggregate >> export >> pre_import
+        if is_release:
```

Comment:
The `counts` tasks above still get defined and run if release is true.  Should those be moved into the else block?

Reply:
That is the intended behaviour. `daily` and `clients` tasks need to run daily for release too, otherwise the single weekly run would have to process the whole week in a day.

Reply:
I'll add a comment there clarifying